components/prism-crystal.js

```diff
@@ -0,0 +1,54 @@
+(function(Prism) {
+	Prism.languages.crystal = Prism.languages.extend('ruby', {
+		number: /\b(?:0b[01_]*[01]|0o[0-7_]*[0-7]|0x[0-9a-fA-F_]*[0-9a-fA-F]|(?:[0-9][0-9_]*[0-9]|[0-9])(?:\.[0-9_]*[0-9])?(?:[eE][+-]?[0-9_]*[0-9])?)(?:_[uif](?:8|16|32|64|))?\b/,
```

Comment:
This part `(?:[0-9][0-9_]*[0-9]|[0-9])` could probably be rewritten as `[0-9](?:[0-9_]*[0-9])?`.
Also, the last group `(?:8|16|32|64|)` might be more explicit written as `(?:8|16|32|64)?`.